pint/default_en.txt

```diff
@@ -467,6 +466,30 @@ buckingham = debye * angstrom
 bohr_magneton = e * hbar / (2 * m_e) = µ_B = mu_B
 nuclear_magneton = e * hbar / (2 * m_p) = µ_N = mu_N
 
+# Logaritmic Unit Definition
+## Unit = scale; logbase; logfactor 
+## x_dB = [logfactor] * log( x_lin / [scale] ) / log( [logbase] )
+ 
+# Logaritmic Units of dimensionless quantity: [ https://en.wikipedia.org/wiki/Level_(logarithmic_quantity) ]
+
+decibell = 1 ; logbase: 10; logfactor: 10 = dB
+# bell = 1 ; logbase: 10; logfactor: = B
+## NOTE: B (Bell) symbol conflicts with byte
+##       Defining dB as deci- Bell causes bad assignment of the unit to ScaleConverter, with poor results
+
+decade = 1 ; logbase: 10; logfactor: 1 
+
+neper = 1 ; logbase: 2.71828182845904523536028747135266249775724709369995; logfactor: 0.5 = Np
```

Comment:
Do not put here the `2.71...`, use the constant defined in `contants`

Reply:
I defined `eulers_number` in `constants` [ [ee0a4cb](https://github.com/hgrecco/pint/pull/1024/commits/ee0a4cb29494d5a4c0453c8785ef992ceeff36f2) ], but parsing constants within the modifier raises an error:
```
AssertionError: 1 != 0 : 1 warnings raised.
E   In line 483, cannot add 'neper = 1 ; logbase: eulers_number; logfactor: 0.5 = Np' Unit definition ('neper') must contain only numbers in modifier, not  eulers_number
```

Reply:
I forgot in the commit the neper's definiton with eulers_number: you can see in Travis that most tests fail :(

Reply:
If magic numbers in the definition are a problem, how would you suggest to proceed? Should I remove neper's definition from `default-en.txt`?